web/src/navigator.rs

```diff
@@ -60,39 +44,30 @@ impl NavigatorBackend for WebNavigatorBackend {
         window_spec: Option<String>,
         vars_method: Option<(NavigationMethod, IndexMap<String, String>)>,
     ) {
-        const JAVASCRIPT_PREFIX: &str = "javascript:";
+        const JAVASCRIPT_PREFIX: &str = "javascript";
```

Comment:
`JAVASCRIPT_PREFIX` is used only once now. I think it's better to just use `"javascript"` directly.